screens/LoginPage.html

```diff
@@ -0,0 +1,76 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://pushpa-ui.netlify.app/documentation/doc.css"
+    />
+    <link rel="stylesheet" href="../components/main.css" />
+    <link rel="stylesheet" href="./autheticate.css" />
```

Comment:
typo in 'authenticate'. You may change the file name

Reply:
This thing a dev can't run from😥